src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonApiService.kt

```diff
@@ -75,21 +78,37 @@ class PrisonApiService(private val prisonWebClient: WebClient) {
       .retrieve()
       .awaitBodilessEntity()
 
-  suspend fun getPrisonerInfo(prisonerNumber: String): PrisonerAtLocation =
-    prisonWebClient.get()
-      .uri("/api/bookings/offenderNo/$prisonerNumber")
-      .retrieve()
-      .awaitBody()
+  suspend fun getPrisonerInfo(prisonerNumber: String, useClientCredentials: Boolean = false): PrisonerAtLocation {
+    return if (useClientCredentials) {
+      prisonWebClientClientCredentials.get()
```

Comment:
These seem to be the same call, but different web clients, be better to use same code, just different variable

Reply:
Good spot! addressed